hack/jenkins/build.sh

```diff
@@ -62,7 +62,7 @@ git clean -fdx
 # docker images -q | xargs -r docker rmi
 
 # Build
-go run ./hack/e2e.go -v --build
+make release
```

Comment:
this will cause PR Jenkins to do a full `make release` too, right? is that what we want?